SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKSimulatedPlayerDeployment.cpp

```diff
@@ -493,7 +493,8 @@ void SSpatialGDKSimulatedPlayerDeployment::OnNumberOfSimulatedPlayersCommited(ui
 
 bool SSpatialGDKSimulatedPlayerDeployment::AttemptSpatialAuth()
 {
-	FString SpatialInfoArgs = TEXT("auth login");
+	bool UseChinaAuth = GetDefault<USpatialGDKEditorSettings>()->GetPrimaryRegionCode().ToString() == TEXT("CN");
+	FString SpatialInfoArgs = UseChinaAuth ? TEXT("auth login --environment=cn-production") : TEXT("auth login");
```

Comment:
Maybe 
```suggestion
	FString SpatialInfoArgs = FString::Printf("auth login %s", UseChinaAuth ? TEXT("--environment=cn-production") : TEXT(""));
```

Reply:
I think the first version is slightly easier to ready? I think complexity is going to be the same or less in first case?